pymisp/mispevent.py

```diff
@@ -629,6 +629,7 @@ def __init__(self, name: str, strict: bool=False, standalone: bool=False, defaul
         self.last_seen: datetime
         self.__fast_attribute_access: dict = defaultdict(list)  # Hashtable object_relation: [attributes]
         self.ObjectReference: List[MISPObjectReference] = []
+        self._standalone: bool = False
```

Comment:
Why is it defined `False` here? while the default (param) is `True` ?

Reply:
The constructor first initializes an object in a coherent state : `__not_jsonable` does not contain "ObjectReference", so it is not standalone. Then, later on, the value of the `standalone` parameter is taken into account (`self.standalone = standalone`), updating `__not_jsonable` if needed